docs/spec_draft.md

```diff
@@ -343,6 +344,78 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_grad
+
+### Semantics
+
+Computes gradients of `Batch Normalizing Transform` for `operand`, `scale` and
```

Comment:
`Batch Normalizing Transform` is the algorithm referred to Algorithm 1 in the paper. If we want to refer to the same algorithm its better to add the link with the algorithm name. The formula mentioned in the paper seems to be different from the one mentioned in HLO spec. Can you please comment on that?

Reply:
We dont want to refer to that algorithm, but the gradient of that algorithm (the differential equations in the paper).

The formulas are different, but they should be equivalent or the back-prop and gradient descent wont work properly.